template/gradle/libs.versions.toml

```diff
@@ -0,0 +1,98 @@
+[versions]
+compileSDK = "33"
```

Comment:
The mix between camelCase and dash-case triggers me just a little bit, but might just need a big of time to get used to it.

Reply:
if I use compile-sdk it will generate compile.sdk. I can however make
sdk-compile
sdk-target
sdk-min

but for androidGradlePlugin I don't think android.gradle.plugin makes sense...